### PR TITLE
goplus 1.0.38

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/goplus.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Goplus < Formula
   desc "Programming language for engineering, STEM education, and data science"
   homepage "https://goplus.org"
-  url "https://github.com/goplus/gop/archive/v1.0.37.tar.gz"
-  sha256 "edbb7eb99f175423d583de156967b5d9ae632b2fe405b6c1a36ede47591d7dd5"
+  url "https://github.com/goplus/gop.git",
+      tag:      "v1.0.38",
+      revision: "8ab2d87ef7948cf9fb66b3edeeb7caa1320db47c"
```
Comment (carlocab) [inline Formula/goplus.rb:6]: Can we just create the `VERSION` file instead of using a Git clone?

Comment (chenrui333) [reply Formula/goplus.rb:6]: I looked at the source file, I dont think it does respect `VERSION` file.

Comment (chenrui333) [reply Formula/goplus.rb:6]: ok, let me give a shot.

Comment (chenrui333) [reply Formula/goplus.rb:6]: I did not get the tarball version working. :(

Receving test failure as below
```
$ /opt/homebrew/Cellar/goplus/1.0.38/bin/gop env GOPVERSION
panic: Invalid buildVersion: 1.0.38

goroutine 1 [running]:
github.com/goplus/gop/env.init.0()
	/private/tmp/goplus-20211228-27233-ign710/gop-1.0.38/env/version.go:35 +0xd4
```

Comment (branchvincent) [reply Formula/goplus.rb:6]: maybe just missing a leading `v`? this worked for me on intel

```ruby
(buildpath/"VERSION").write "v#{version}"
```

**Review by branchv — APPROVED** [review]

**Review by carlocab — APPROVED** [review]

File: Formula/goplus.rb [hunk 2]
```diff
@@ -20,6 +19,7 @@ class Goplus < Formula
 
   def install
     ENV["GOPROOT_FINAL"] = libexec
+    (buildpath/"VERSION").write "v#{version}"
```
Comment (chenrui333) [inline Formula/goplus.rb:22]: Oh, I missed `v` 🤦 

Thanks @branchvincent!

Comment (branchvincent) [reply Formula/goplus.rb:22]: anytime!

**Pull request closed without merging**